src/js/common/web3.js

```diff
@@ -58,3 +58,7 @@ export async function requestPermission() {
 export async function verifyChainId(chainId) {
   return (await web3.eth.getChainId()) === parseInt(chainId)
 }
+
+export async function getDefaultAccount() {
+  return (await web3.eth.getAccounts())[0]
```

Comment:
This can return undefined if the wallet is locked, we need to handle that somewhere.